setup.py

```diff
@@ -345,6 +345,7 @@ def main():
             "include/nvfuser/kernel_db/*.h",
             "include/nvfuser/multidevice/*.h",
             "include/nvfuser/ops/*.h",
+            "include/nvfuser/ir/*.h",
```

Comment:
I assume including extra headers doesn't hurt? Do we need to try a !build here?

Reply:
Yeah we definitely need a `!build`